app/src/main/java/org/mozilla/fenix/settings/wallpaper/Extensions.kt

```diff
@@ -25,3 +28,37 @@ fun List<Wallpaper>.groupByDisplayableCollection(): Map<Wallpaper.Collection, Li
 }.toMap().takeIf {
     it.isNotEmpty()
 } ?: mapOf(Wallpaper.DefaultCollection to listOf(Wallpaper.Default))
+
+/**
+ * The ideal scenario is to have 3 seasonal wallpapers and 2 classic, minding the order.
+ * But in case there are less than 3 or no seasonal wallpapers at all, they are replaced by classic.
+ * Edge cases where there are less then 6 items overall would return less items in the right order.
+ */
+fun List<Wallpaper>.groupForWallpaperOnboarding(): List<Wallpaper> {
```

Comment:
```suggestion
fun List<Wallpaper>.getWallpapersForOnboarding(): List<Wallpaper> {
```
We aren't grouping anything like a `Map` so I think we shouldn't use the same naming as above.